app/src/main/java/com/sunasterisk/getitdone/ui/home/TaskAdapter.kt

```diff
@@ -123,4 +124,12 @@ class TaskAdapter : BaseAdapter<Task, TaskAdapter.TaskViewHolder>() {
             }
         }
     }
+
+    class HeaderViewHolder(itemView: View, override var clickItemListener: (String) -> Unit) :
+        BaseViewHolder<String>(itemView) {
+        override fun bindData(item: String) {
+            super.bindData(item)
+            itemView.text_completed_task_title.text = item
```

Comment:
```suggestion
            itemView.textCompletedTaskTitle.text = item
```
Vì đang dùng `android kotlin extension`